history.py

```diff
@@ -0,0 +1,31 @@
+from django_mako_plus import view_function, jscontext

```

Comment:
that a legit way of making the last 5 class

Reply:
Appreciate it! Thanks.